### PR TITLE
fix: Don't publish comments when no files is update

### LINKED ISSUE #534: "Publish PR preview" comments: don't include extra files in the list of pages
### 1st use case: only update navigation files

This has been detected during implementation of  #516 to fix  #511.
It was reproduced with https://github.com/bonitasoft/bonita-central-doc/pull/7 and https://github.com/bonitasoft/bonita-doc/pull/2318

When navigation files change, they are present in the list of URL to check, in the PR comment.
However, this file are not generated as standalone pages, so the links lead to a HTTP 404.
So they should remove from this list.

ℹ️ Possible future improvements: #677

### 2nd use case: only update the workflow generating the preview

An extra undefined/undefined link is created.
See https://github.com/bonitasoft/bonita-doc/pull/2417

image

### 3rd use case: only update AsciiDoc attributes

Update the antora.yml file
An extra undefined/undefined link is created.
See https://github.com/bonitasoft/bonita-doc/pull/2415

### PR DESCRIPTION
**Pull request by benjaminParisel**
Skip the Comments step when no file updates are detected.

Closes #534

- [x] :warning:  Before Merge this PR, we need to remove the cherry-picked commit

### CODE REVIEW

**Comment by benjaminParisel** [comment]
I cherry-picked the commit from the #673 Pull request to test the preview workflow.
In #673, we see a useless comment with some `undefined/undefined` links.

image

With the fix in this PR, this link are skipped.

**Review by tbouffard — APPROVED** [review]
good job!

**Comment by tbouffard** [comment]
Put in draft until we remove the cherry-pick commit to validate the fix.
Once merged, this will be tested in #673

File: .github/actions/comment-pr-with-links/action.yml [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
+    description: "The name of the component to build. If set, the build-preview-command input is ignored"
     required: false
-    default: 'bonita'
+    default: "bonita"
   pattern:
-    description: ''
+    description: ""
     required: false
-    default: 'modules/**/*.adoc'
+    default: "modules/**/*.adoc"
```
[the remark this answers is not in the document]
Comment (benjaminParisel) [reply .github/actions/comment-pr-with-links/action.yml:15]: We need to try i think

[the remark this answers is not in the document]
Comment (tbouffard) [reply .github/actions/comment-pr-with-links/action.yml:15]: Fine, I am going to open a fake PR in the labs-doc repository.

Comment (tbouffard) [inline .github/actions/comment-pr-with-links/action.yml:15]:
```suggestion
    default: "modules/**/pages/**/*.adoc"
```

[the remark this answers is not in the document]
Comment (tbouffard) [reply .github/actions/comment-pr-with-links/action.yml]: Commited in aaf1a82.
✔️ Tested in https://github.com/bonitasoft/bonita-labs-doc/pull/152#issuecomment-1961361859. The previous edit shows that the navbar files was previously listed. This is not the case anymore.

**Comment by tbouffard** [comment]
I am merging this PR and validate the change in the master branch by updating https://github.com/bonitasoft/bonita-labs-doc/pull/152

**Pull request merged**